tests/test_http.py

```diff
@@ -26,6 +27,11 @@ def s():
 
 def has_connection():
     # In case we run tests in an environment without internet connection.
+
+    if sys.version_info < (2, 7):
+        # py 2.6 doesn't have SNI support, required for httpbin, as it has SSLv3 certificates
+        return False
```

Comment:
Reverted that commit, tests still fail: https://travis-ci.org/projectatomic/osbs-client/builds/235968189?utm_source=github_status&utm_medium=notification

>We weren't seeing "hostname doesn't match" errors -- we were seeing an "internal error".

I believe this is hidden in this `SSLError`